test/TestLogger.UnitTests/TestResultInfoTests.cs

```diff
@@ -14,28 +14,24 @@ public class TestResultInfoTests
         [TestMethod]
         public void GetHashCodeShouldReturnAnUniqueHash()
         {
-            var result = new TestResult(new TestCase());
-            var resultInfo = new TestResultInfo(result, string.Empty, string.Empty, string.Empty);
+            var resultInfo = new TestResultInfoBuilder(string.Empty, string.Empty, string.Empty).Build();
 
             Assert.AreNotEqual(new TestResult(new TestCase()).GetHashCode(), resultInfo.GetHashCode());
         }
 
         [TestMethod]
         public void EqualsShouldReturnFalseForNonTestResultInfoObject()
         {
-            var result = new TestResult(new TestCase());
-            var resultInfo = new TestResultInfo(result, string.Empty, string.Empty, string.Empty);
+            var resultInfo = new TestResultInfoBuilder(string.Empty, string.Empty, string.Empty).Build();
 
             Assert.IsFalse(resultInfo.Equals(new ()));
         }
 
         [TestMethod]
         public void EqualsShouldReturnFalseIfErrorMessageOrStackTraceDoNotMatch()
         {
-            var result1 = new TestResult(new TestCase()) { ErrorMessage = "error 1" };
-            var result2 = new TestResult(new TestCase()) { ErrorMessage = "error 2" };
-            var r1 = new TestResultInfo(result1, string.Empty, string.Empty, string.Empty);
-            var r2 = new TestResultInfo(result2, string.Empty, string.Empty, string.Empty);
+            var r1 = new TestResultInfoBuilder(string.Empty, string.Empty, string.Empty).WithErrorMessage("error 1").Build();
+            var r2 = new TestResultInfoBuilder(string.Empty, string.Empty, string.Empty).WithErrorMessage("error 1").Build();
 
             Assert.IsFalse(r1.Equals(r2));
```

Comment:
I don't understand why the TestResultInfo.Equals is supposed to check equality on only the error message and stack trace.

Reply:
I don't recollect the context. Do we use result equality comparison in our loggers?

Reply:
There weren't any references to .Equals so I got rid of what looked like the bad equality check

Reply:
Sounds good, our tests should catch if there's any regression here.